h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTreeMojoConverterTest.java

```diff
@@ -0,0 +1,170 @@
+package hex.tree.xgboost;
+
+import biz.k11i.xgboost.Predictor;
+import biz.k11i.xgboost.gbm.GBTree;
+import biz.k11i.xgboost.tree.RegTree;
+import biz.k11i.xgboost.tree.RegTreeNode;
+import hex.genmodel.algos.tree.SharedTreeGraph;
+import hex.genmodel.algos.tree.SharedTreeSubgraph;
+import hex.genmodel.algos.xgboost.XGBoostJavaMojoModel;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import water.*;
+import water.fvec.Frame;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(Parameterized.class)
+public class XGBoostTreeMojoConverterTest extends TestUtil {
+
+    @Parameterized.Parameters(name = "{index}: gbm({0})")
+    public static Iterable<?> data() { 
+        return Arrays.asList(
+                0,  // first tree 
+                -1  // all trees
+        );
+    }
+
+    @Parameterized.Parameter
+    public int _treeToPrint;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void stall() {
+        stall_till_cloudsize(1);
+
+        // we need to check for XGBoost backend availability after H2O is initialized, since we
+        // XGBoost is a core extension and they are registered as part of the H2O's class main method
```

Comment:
this part `... initialized, since we XGBoost is a core extension ...` has extra `we`

Reply:
thank you @koniecsveta, fixed